Unit2/Module2.2/Mini-Project2.2a/README.md

```diff
@@ -0,0 +1,18 @@
+# Contribute: Open Source Project
+
+Git is a very powerful tool used around the world that allows several individuals to work together on the same project at the same time! Pretty cool, huh?
+
+Let's see what its like to use the collaborative tool GitHub to work on an open source project.
+
+1. Head back over to the OpenClassrooms **Git and GitHub: Part 1 #6** course page [here](https://openclassrooms.com/en/courses/5671626-manage-your-code-project-with-git-github/6152046-contribute-to-an-open-source-project-with-basic-commands-on-git).
+1. Follow the instructions on this page very carefully.
+1. Submit the PR as described in the OC course link.
+1. Take a screenshot of the PR page you just created on GitHub.
+    1. Add and commit it to this directory on your local computer as `./screenshot.png`
+1. Copy the PR link and edit this `README.md` below by pasting the link.
+1. Of course, save and commit your changes :)
+
+## Proof of Pull Request
```

Comment:
In order to avoid annoying/confusing behavior with Git Subtree, I'm thinking we should generally avoid having students edit files within the course repo directly. Rather, I suggest as a general rule they would always upload their own files. So instead of saying "Edit this file", we could say, copy this file, edit it and save it in the project directory (as a new file).

What do you think?